src/pages/library/dataBrowser-utils.js

```diff
@@ -57,6 +57,12 @@ const normalizeSnapshot = snapshot => {
     _.uniqBy(_.toLower)
   )(snapshot['prov:wasGeneratedBy'])
 
+  //  For beta test: if the title length is even, pretend that this is a controlled snapshot.
+  if (snapshot['dct:title'].length % 2 === 0) {
+    snapshot.roles = ['discoverer']
+    snapshot['TerraDCAT_ap:hasDataUsePermission'] = 'TerraCore:CC'
+  }
+
```

Comment:
We typically try to avoid mutation in the code - you could do this and put it as the first line in the function:

```suggestion
  //  For beta test: if the title length is even, pretend that this is a controlled snapshot.
  const snapshot = rawSnapshot['dct:title'].length % 2 === 0 ? {
    ...rawSnapshot,
    roles: ['discoverer'],
    'TerraDCAT_ap:hasDataUsePermission': 'TerraCore:CC'
  } : rawSnapshot

```

`rawSnapshot` will be the new name of the input parameter of the function
I really like using the title length to introduce randomness!